docs/src/pages/ui/primitives/phonenumberfield/react.mdx

```diff
@@ -0,0 +1,29 @@
+import { TextField, Flex, PasswordField, Button } from '@aws-amplify/ui-react';
+import { PhoneNumberFieldDemo } from './demo';
+import { Example } from '@/components/Example';
+
+The `PhoneNumberField` form primitive can be used allow users to input a phone number.
+
+## Demo
+
+<PhoneNumberFieldDemo />
+
+## Usage
+
+### Autocomplete - supporting password managers
+
+Password managers in Chromium expect input forms to have an `autocomplete` attribute of `username`. As a result, this attribute is
+added to the `PhoneNumberField` primitive so that password managers are supported out of the box in Chromium. However, you may pass in
+your own `autocomplete` attribute if you wish to override the default value.
+
+See https://www.chromium.org/developers/design-documents/form-styles-that-chromium-understands for more information.
+
+### Additional Usage
+
+`PhoneNumberField` is a custom primitive built on top of the `TextField` primitive. Refer to the [TextField primitive documentation](textfield#usage)
+for more information regarding additional usage.
+
+## Styling
+
+The `PhoneNumberField` uses the same styling rules as the `TextField` primitive. Refer to the [TextField primitive documentation](textfield#css-styling)
```

Comment:
Our documentation strategy for primitives is to lean towards being repetitive rather than linking to other primitives. We feel like users will want to know everything about a primitives from a single page. Please add examples of use of `size`, `variation`, and CSS Styling. I think we also need to record the `onChange` and `onInput` event handler use mentioned in previous comment and an example of using `defaultCountryCode` See https://quip-amazon.com/fOKyAeLgbNCa/Template-for-Primitive-Docs